src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -74,6 +77,7 @@ const ReportActionContextMenu = (props) => {
                     text={contextAction.text}
                     isMini={props.isMini}
                     key={contextAction.text}
+                    onPress={contextAction.text === 'Mark as Unread' ? props.onMarkAsUnread : () => {}}
```

Comment:
`Mark as Unread` should be using a constant if we are going to string comparison check like this that way if someone changes the text they don't have to remember updating this here.